src/main/resources/org/rust/ide/liveTemplates/Rust.xml

```diff
@@ -60,4 +60,28 @@
         </context>
     </template>
 
+    <template name="loop" description="Infinite loop" toReformat="true" toShortenFQNames="true"
+              value="loop {&#10;  $SELECTION$&#10;}$END$">
```

Comment:
it doesn't really matter because IntelliJ will automatically reformat the snippet according to the code formatter settings